note.py

```diff
@@ -159,27 +170,55 @@ def __note_filename(self, name):
         return os.path.join(self.__notespath, name, "note.md")
 
     def geometry(self, geometry=None):
-        """Returns the geometry (size) of the application window"""
+        """
+        Returns and optionally sets the geometry (size)
```

Comment:
```suggestion
        Returns and optionally sets the geometry
```
it saves size and position